nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -317,6 +318,17 @@ class RetentionAnalytics extends mixinBehaviors([I18nBehavior, FormatBehavior],
     };
   }
 
+  static get observers() {
+    return ['_resetChartData(visible)'];
+  }
+
+  _resetChartData(visible) {
+    if (visible) {
+      const repositories = this.shadowRoot.querySelectorAll('nuxeo-repository-data');
+      repositories.forEach((repository) => repository.ready());
```

Comment:
Even it works fine, I think it will better to get the feedback on it from the UI team :) 
@semisse  @nelsonsilva  wdyt about the usage of `this.shadowRoot.querySelectorAll`

Reply:
I think it's fine, no need to explicitly get every single element by id bu shouldn't we use `repository.fetch()` instead?
FYI I think we've done something similar in other places just by wrapping the content in a `<dom-if if="[[visible]]" restamp>..</dom-if>` which effective cause everything in the DOM to be restamped, still your approach is nicr as we just update the data.

Reply:
@nelsonsilva thanks a lot for the feedback! I made the change, does that look good? Thanks!